crates/dbt-jinja/minijinja/src/value/mod.rs

```diff
@@ -747,6 +747,12 @@ impl Value {
     /// and this is the only way to construct it.
     pub const UNDEFINED: Value = Value(ValueRepr::Undefined);
 
+    /// The none value.
+    ///
+    /// This constant exists because the none type does not exist in Rust
+    /// and this is the only way to construct it.
+    pub const NONE: Value = Value(ValueRepr::None);
```

Comment:
`minijinja` uses `Value::from(())` throughout its code. But a constant is welcome and more discoverable.